.repo-metadata.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "gce-images",
+  "name_pretty": "GCE Images",
+  "product_documentation": "https://cloud.google.com/compute/docs/images",
+  "client_documentation": "https://googleapis.dev/nodejs/gce-images/latest",
+  "issue_tracker": "https://github.com/googleapis/gce-images/issues",
+  "release_level": "GA",
```

Comment:
don't know that it matters, but I feel like we've been using lowercase `beta`, `alpha`, `ga`.

Reply:
https://github.com/googleapis/nodejs-storage/blob/master/.repo-metadata.json#L7